### ISSUE TITLE
tables with a size which is not a power of two?

### ISSUE THREAD

**Issue by tjingboem** [csound/manual#706]
https://csound.com/manual/SiggenTableacc.html

Is this Note no longer true for 7.0?
[Note]	Note
Although tables with a size which is not a power of two can be created using a negative size (see [f score statement](https://csound.com/manual/f.html)), some opcodes will not accept them.

**Comment by vlazzarini** [comment]
no longer true. Also there is no need to use negative size since 6.0.

**Comment by tjingboem** [comment]
good. i will remove this Note.
Can you review https://csound.com/manual/f.html and see if this information is still correct in this respect?

**Comment by joachimheintz** [comment]
yes but i suggest that on this manual page we should mention the issues 
we discussed about the extended guard point and its effect on the 
reported table size.  perhaps victor can say whether this is correct and 
can go into the manual:

Note that setting a *positive* table size as a power of two plus one is 
interpreted by Csound as a power of two size with extended guard point 
for interpolation.
E.g. positive table size 8 and 9 both will report 8 as actual table size:
	ftlen(ftgen(0,0,8,2,0)) -> 8
	ftlen(ftgen(0,0,9,2,0)) -> 8

A *negative* table size however will set the real size of the table, 
without distiguishing between power-of-two-plus-one and other sizes:
	ftlen(ftgen(0,0,-8,2,0)) -> 8
	ftlen(ftgen(0,0,-9,2,0)) -> 9

**Comment by tjingboem** [comment]
GEN20 still Must be a power of 2 ( + 1).
